lib/Runtime/ByteCode/ByteCodeEmitter.cpp

```diff
@@ -3809,6 +3826,7 @@ void ByteCodeGenerator::StartEmitFunction(ParseNode *pnodeFnc)
                 !funcInfo->Escapes() &&
                 funcInfo->frameObjRegister != Js::Constants::NoRegister &&
                 !ApplyEnclosesArgs(pnodeFnc, this) &&
+                funcInfo->paramScope->GetCanMergeWithBodyScope() && // There is eval in the param scope
```

Comment:
Actually, on second thought, that might be a bit hard because we have the notion of CallsEval but not at the scope level. Nevermind.